CONTRIBUTING.md

```diff
@@ -14,11 +15,17 @@ In addition to everything written in here, there is a [Collaborator Guide](https
 * [Commit Messages](#commit-messages)
 
 ## Process
-
-1. Clone this repository
-2. Create a branch for your feature (see [below](#feature-branch-example) for an example)
-3. Add your code
-4. Create a Pull Request
+### Initial Setup
+1. Fork the repository
+2. Clone your forked repository onto your local machine
+3. inside your CLI (Command Line Interface), move into your working directory
+4. run `npm i` inside the root, client *and* server folder to install all dependencies needed for this project
+5. inside the client folder, run `npm run dev` to start the development server
+
+### How to Contribute
+1. Create a branch for your feature (see [below](#feature-branch-example) for an example)
```

Comment:
"#feature-branch-example" should probably be named to "feature/#issuenumber-featuredescription"